testdata/generate_test_data/test_circles_onto.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$schema": "../../src/dsp_tools/resources/schema/project.json",
+    "project": {
+        "shortcode": "4123",
+        "shortname": "systematic-tp",
```

Comment:
to prevent name clashes with our current e2e tests, I suggest taking another shortcode / shortname that is not used yet in any JSON file in the DSP-TOOLS repo

Reply:
is there a shortcode you recommend?

Reply:
No, just take a hexadecimal number with four digits, and quickly search in the repo that it is not used yet.